Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -148,12 +150,24 @@ def __init__(self, url: str, username: str, password: str, token: str | None = N
 
     def _http_request(self, **kwargs):
```

Comment:
I think we should add more logs here to see the raised error in the logs when iterating.

Reply:
I print it in line 160, do you think I should add any additional log?